index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Trail-Project</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+        <div id="page-wrapper">
+          <header id="header">
+            <div class="logo">
+              <img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" alt="original trombones logo">
+            </div>
+    
+            <nav id="nav-bar">
+              <ul>
+                <li><a class="nav-link" href="#features">Features</a></li>
+                <li><a class="nav-link" href="#how-it-works">How It Works</a></li>
+                <li><a class="nav-link" href="#pricing">Pricing</a></li>
+              </ul>
+            </nav>
+          </header>
+    
+          <div class="container"></div>
+    
+          <section id="hero">
+            <h2>Handcrafted, home-made masterpieces</h2>
+            <form id="form" action="https://www.freecodecamp.com/email-submit">
+              <input name="email" id="email" type="email" placeholder="Enter your email address" required="">
+              <input id="submit" type="submit" value="Get Started" class="btn">
+            </form>
+          </section>
```

Comment:
- Please add a label tag. Label tag make the site more readable and accessible for screen readers.